tests/foreman/ui_airgun/test_os.py

```diff
@@ -1,11 +1,13 @@
 from nailgun import entities
 
-from robottelo.datafactory import gen_string, valid_data_list
-from robottelo.decorators import parametrize
+from robottelo.datafactory import gen_string
 
 
-@parametrize('name', valid_data_list())
-def test_positive_create(session, name):
+# Next fixture can be used to test different data sets
+# @parametrize('name', valid_data_list())
```

Comment:
Why commenting? Just to avoid utf8? How about commenting utf8 in data factory instead (or rather generating some more BMP-only utf8). Parametrize would be great for demo purposes.

Reply:
and what parametrize will show to anyone? but okay =))